src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Homework19 extends BaseTest {
+    @Test(dataProvider = "invalidCredentials", dataProviderClass = BaseTest.class)
+    public void deletePlaylist() {
+        provideEmail("denise626@hotmail.com");
+        providePassword();
+        clickSubmitBtn();
+
+        WebElement choosePlaylist = driver.findElement(By.cssSelector("[href = '#!/playlist/17356']"));
+        choosePlaylist.click();
+        WebElement deleteSongPlaylist = driver.findElement(By.cssSelector("[id='extraTabAlbum']"));
```

Comment:
There is a button to delete "Delete this playlist"

Locator `//button[@title='Delete this playlist']`

![image](https://user-images.githubusercontent.com/104761482/198518107-ed577fb8-b625-4aaf-91ff-4377d1324de5.png)